### PR TITLE
feat(o-shop): revert readme and more cleanup

### PR DESCRIPTION
**Pull request by coffee-intocode**

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    <meta charset="utf-8" />
+    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta name="theme-color" content="#000000" />
+    <meta
+      name="description"
+      content="Web site created using create-react-app"
+    />
+    <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
+    <!--
```
Comment (ahaly92) [inline public/index.html]: remove comments in this file. They dont really serve a purpose and make it hard to see what's happening here

File: public/manifest.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
+{
+  "short_name": "React App",
+  "name": "Create React App Sample",
+  "icons": [
+    {
```
Comment (ahaly92) [inline public/manifest.json]: let's get rid of all these images since we dont really need em

File: src/App.css [hunk 3]
```diff
@@ -0,0 +1,38 @@
+.App {
+  text-align: center;
+}
+
+.App-logo {
```
Comment (ahaly92) [inline src/App.css]: lets delete everything from line down

File: src/App.js [hunk 4]
```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import logo from './logo.svg';
+import './App.css';
+
+function App() {
+  return (
+    <div className="App">
```
Comment (ahaly92) [inline src/App.js]: let's simplify this file file to just say <h1>App</h1>

File: src/index.js [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+import App from './App';
+import * as serviceWorker from './serviceWorker';
+
+ReactDOM.render(
+  <React.StrictMode>
+    <App />
+  </React.StrictMode>,
+  document.getElementById('root')
+);
+
```
Comment (ahaly92) [inline src/index.js:13]: let's remove comments in this file

File: src/logo.svg [hunk 6]
```diff
@@ -0,0 +1,7 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 841.9 595.3">
```
Comment (ahaly92) [inline src/logo.svg]: let's delete this image to clean this up

File: src/serviceWorker.js [hunk 7]
```diff
@@ -0,0 +1,141 @@
+// This optional code is used to register a service worker.
```
Comment (ahaly92) [inline src/serviceWorker.js]: remove comments here

File: src/setupTests.js [hunk 8]
```diff
@@ -0,0 +1,5 @@
+// jest-dom adds custom jest matchers for asserting on DOM nodes.
```
Comment (ahaly92) [inline src/setupTests.js]: remove comments

**Comment by coffee-intocode** [comment]
@ahaly92 please review when you get a second.

**Comment by ahaly92** [comment]
hey everything looks good, all you need to do is remove the pngs and svgs and you should be good to go

**Pull request merged**